pkg/validation/runner_test.go

```diff
@@ -0,0 +1,126 @@
+package validation_test
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	"github.com/aws/eks-anywhere/internal/test"
+	anywherev1 "github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	eksaerrors "github.com/aws/eks-anywhere/pkg/errors"
+	"github.com/aws/eks-anywhere/pkg/validation"
+)
+
+func TestRunnerRuanAllSuccess(t *testing.T) {
+	g := NewWithT(t)
+	ctx := context.Background()
+	r := validation.NewRunner[*cluster.Spec]()
```

Comment:
Could we avoid `cluster.Spec` and use a test object instead to decouple them from `cluster.Spec` changes?

Reply:
Yes sir